server/aws/nodes.yaml

```diff
@@ -336,11 +343,17 @@ Resources:
             00_deploy:
               env:
                 BRANCH: !Ref 'Branch'
+                DAEMONOPTS: !Join [' ', [
+                  '--no-igd',
+                  '--hide-my-port',
```

Comment:
I thought that this would be used for trusted nodes as well. Maybe make it optional?

Reply:
Which option makes it not usable as a trusted node? Or is it both of them?

Reply:
It just doesn't make too much sense to try hiding when we are actually open to the world on purpose; although, for a node that isn't published anywhere (for example vpn without trusted), it is correct as it is. Hence why I asked if it can be optional.